src/GLPI.php

```diff
@@ -33,15 +33,9 @@
  * ---------------------------------------------------------------------
  */
 
-use Glpi\Application\ErrorHandler;
-use Monolog\Formatter\LineFormatter;
-use Monolog\Handler\StreamHandler;
-use Monolog\Logger;
-use Psr\Log\LogLevel;
-
 /**
- *  GLPI (instantiation and so on)
- **/
+ * @FIXME Find a better place for these constants.
```

Comment:
Which constants are you talking about ?

Reply:
This class only contains constants now.

Reply:
I see. Well I guess we can just rename it and turn it into an enum.